sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -64,8 +64,18 @@ internal async ValueTask<int> AddBatchActivityAsync(IEnumerable<Activity> batchA
                 telemetryItems.Add(telemetryItem);
             }
 
+            Azure.Response<TrackResponse> response;
+
+            if (async)
```

Comment:
Curious - why would we need this branching logic?

Reply:
:) Azure SDK forces to pass async bool passed as function argument when we don't `await` and use `GetResult` (EnsureCompleted())